rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.stats/70_write_load.yml

```
@@ -0,0 +1,59 @@
+---
+setup:
+  - skip:
+      version: " - 8.5.99"
+      reason: Indices write load stats were introduced in 8.6
+---
+"Write load average is tracked at shard level":
+  - do:
+      indices.create:
+        index: testindex
+        body:
+          settings:
+            index.number_of_shards: 1
+            index.number_of_replicas: 0
+          mappings:
+            properties:
+              name:
+                type: text
+              description:
+                type: text
+              price:
+                type: double
+
+  - do:
+      indices.stats:
+        index: "testindex"
+        level: shards
+        metric: [ indexing ]
+
+  - match: { _all.total.indexing.write_load: 0.0 }
+  - match: { indices.testindex.total.indexing.write_load: 0.0 }
+  - match: { indices.testindex.shards.0.0.indexing.write_load: 0.0 }
+
+  - do:
+      index:
+        index: testindex
+        body: { "name": "specialty coffee", "description": "arabica coffee beans", "price": 100 }
+  - do:
+      index:
+        index: testindex
+        body: { "name": "commercial coffee", "description": "robusta coffee beans", "price": 50 }
+  - do:
+      index:
+        index: testindex
+        body: { "name": "raw coffee", "description": "colombian coffee beans", "price": 25 }
+  - do:
+      index:
+        index: testindex
+        body: { "name": "book", "description": "some book", "price": 1000 }
+
+  - do:
+      indices.stats:
+        index: "testindex"
+        level: shards
+        metric: [ indexing ]
+
+  - gt: { _all.total.indexing.write_load: 0.0 }
+  - gt: { indices.testindex.total.indexing.write_load: 0.0 }
+  - gt: { indices.testindex.shards.0.0.indexing.write_load: 0.0 }
```

I wonder how we are sure of this? Seems to rely on the precision of nano-time being good enough that the few indexing ops will be measurable. Probably true on most platforms, but the guarantee is only that the precision is at least as good as `currentTimeMillis`, which is quite coarse on some platforms.

I think we could probably just verify the format here, i.e., change this to:

```suggestion
  - gte: { _all.total.indexing.write_load: 0.0 }
  - gte: { indices.testindex.total.indexing.write_load: 0.0 }
  - gte: { indices.testindex.shards.0.0.indexing.write_load: 0.0 }
```

and then rely on the verification you have in `IndexShardTests` for the actual value.